src/index.html

```diff
@@ -14,33 +14,43 @@
           <img src="images/Callmar.png" alt="picture of squid" />
         </div>
         <div class="header__btns">
-            <button type="button" class="header__button button">Вход</button>
-            <button type="button" class="header__button-outline button-outline">Регистрация</button>
+          <button type="button" class="header__button button">Вход</button>
+          <button type="button" class="header__button-outline button-outline">Регистрация</button>
+        </div>
+        <div class="header__hamburger-wrap header__hamburger_active">
```

Comment:
An element modifier cannot be above an element. It must be next to the element or the modifier is set to the wrong class